content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling
+date: 2023-10-05T17:03:36.464Z
+author: Naveen Gupta
+authorimage: /img/img-2142.jpg
+thumbnailimage: ""
+disable: false
+---
+Monitoring data center infrastructure is critical for ensuring optimal performance, resource utilization, and timely issue detection. Redfish, an open industry standard for hardware management, provides a standardized way to access telemetry data from servers and other devices. Coupling Redfish telemetry with cloud-native monitoring tools offers a robust solution for real-time monitoring, data analysis, visualization, and alerting. 
+
+## Why does data center monitoring matter?
+
+Data centers are the backbone of modern businesses, housing critical applications, databases, and services. Ensuring the seamless operation of these data centers is essential for business continuity and meeting the demands of today's digital world. Data center monitoring helps with following:
+
+* **Performance Optimization:** Monitoring helps identify bottlenecks, inefficiencies, and potential failures, allowing for proactive optimization.
```

Comment:
it might be worse trying to insert the following style in your markdown at the beginning of your entry (in the CMS editor, toggle on Markdown mode) to format the bullet list to standard text size as explained in our guide here:

https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#tips-and-trick-using-the-cms-editor

(see tips 16).


<style>
li {
   font-size: 27px;
   line-height: 33px;
   max-width: none;
}
</style>

Reply:
@choukden When I add `<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>`, it makes bullets bigger, but rest of the text looks smaller in comparison to bullets. Do you have any suggestion for paragraph's font size?

Reply:
@gupta-naveen 

Yes this is the rendering you can see in the CMS editor on the right side. However, once you apply the style in the markdown, if you look at the preview, I think the bullet items should be the same size as the rest of the content.

Preview: https://deploy-preview-2108--hpe-dev-portal.netlify.app/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling/ 

You can make a try if you wish, or just leave the bullet items as they are now.

Denis

Reply:
Thank you, @choukden for pointing me to preview. Fixed.

Reply:
@gupta-naveen 

You are welcome!
Glad to see it works in the preview.